src/components/withKeyboardState.tsx

```diff
@@ -8,27 +8,34 @@ import type ChildrenProps from '@src/types/utils/ChildrenProps';
 type KeyboardStateContextValue = {
     /** Whether the keyboard is open */
     isKeyboardShown: boolean;
+
+    /** Height of the keyboard in pixels */
+    keyboardHeight: number;
 };
 
 // TODO: Remove - left for backwards compatibility with existing components (https://github.com/Expensify/App/issues/25151)
 const keyboardStatePropTypes = {
     /** Whether the keyboard is open */
     isKeyboardShown: PropTypes.bool.isRequired,
+
+    /** Height of the keyboard in pixels */
+    keyboardHeight: PropTypes.number.isRequired,
 };
 
 const KeyboardStateContext = createContext<KeyboardStateContextValue>({
     isKeyboardShown: false,
+    keyboardHeight: 0,
 });
 
 function KeyboardStateProvider({children}: ChildrenProps): ReactElement | null {
-    const [isKeyboardShown, setIsKeyboardShown] = useState(false);
+    const [keyboardHeight, setKeyboardHeight] = useState(0);
 
     useEffect(() => {
-        const keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', () => {
-            setIsKeyboardShown(true);
+        const keyboardDidShowListener = Keyboard.addListener('keyboardDidShow', (e) => {
+            setKeyboardHeight(e.endCoordinates.height);
```

Comment:
Note that this is not a continuous event. If you open the keyboard this callback is called once. That's just how React Native's built-in `Keyboard` lib works.